components/style/values/specified/svg.rs

```diff
@@ -260,3 +261,17 @@ impl ToCss for SVGPaintOrder {
         Ok(())
     }
 }
+
+/// Specified MozContextProperties value.
+/// Nonstandard (https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-context-properties)
+
+impl Parse for MozContextProperties {
+    fn parse<'i, 't>(
+        _context: &ParserContext,
+        input: &mut Parser<'i, 't>
+    ) -> Result<MozContextProperties, ParseError<'i>> {
+        let location = input.current_source_location();
+        let i = input.expect_ident()?;
+        MozContextProperties::from_ident(location, i, &["all", "none", "auto"])
```

Comment:
Let's do instead:

```rust
Ok(MozContextProperties(CustomIdent::from_ident(..)?))
```

That way you don't need to duplicate `CustomIdent::from_ident`.

Reply:
Aaaah! Didn't think of that way. That's clever :).

I think everything should be ok now. Have a look and I'll `squash` and `push`.